lib/clearance/engine.rb

```diff
@@ -1,12 +1,19 @@
-require 'clearance'
-require 'rails'
+require "clearance"
+require "rails"
 
 module Clearance
   class Engine < Rails::Engine
-    initializer 'clearance.filter' do |app|
+    initializer "clearance.filter" do |app|
       app.config.filter_parameters += [:password, :token]
     end
 
-    config.app_middleware.insert_after ActionDispatch::ParamsParser, Clearance::RackSession
+    config.app_middleware.insert_after(
+      ActionDispatch::ParamsParser,
+      Clearance::RackSession
```

Comment:
My current client uses Rubocop in their codebase and the conventions they have enforce trailing commas in hashes, arrays, and method args. But to each his own!